Components/Avatar/Avatar.css

```diff
@@ -0,0 +1,95 @@
+.small-avatar{
+    height: 7rem;
+    width: 7rem;
+    border-radius: 10rem;
+    margin: 1rem;
+}
+.medium-avatar{
+    height: 14rem;
+    width: 14rem;
+    border-radius: 10rem;
+    margin: 1rem;
+}
+.large-avatar{
+    height: 20rem;
+    width: 20rem;
+    border-radius: 10rem;
+    margin: 1rem;
+}
+
+.avatar-type{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--landing-hover-color);
+}
+
+iframe{
```

Comment:
For responsiveness, you can add max-width: 100% for carbon container

Reply:
added the change